### PR TITLE
Navigation&Home

### PR DESCRIPTION
**Pull request by MariaRiosNavarro**
Please Check:

-correction old recomendations.
-add new assets for Navigation Bar and for Home Page
-Create a small StyledAppContainer to reuse in Navigation & Home
-Create Navigation with the StyledAppContainer and use it at Homepage, Contacts and ContactDetail
-Create a "fake" App Grid for the Icons at home
-Mark the applications that are not yet active (for the future after Bootcamp) in light grey for those of you who correct, so you know what works

[Deployment](https://mobile-simulation-prop-56isrga9r-mariariosnavarro.vercel.app/)

### CODE REVIEW

File: components/StyledAppContainer/index.js
Comment (nastiche) [inline components/StyledAppContainer/index.js:1]: May I ask why do you use here span as container and not div? Just wondering...

File: pages/contacts.js
Comment (nastiche) [inline pages/contacts.js:1]: Do you plan to move your Layout to a component and place it in _app.js?
Screenshot 2023-06-23 at 10 29 54 1

Comment (nastiche) [inline pages/contacts.js:1]: What does StyledHR stand for? Maybe it makes sense to write the whole name for more clearity?

Comment (nastiche) [reply pages/contacts.js:1]: Is it necessary to write return in the map method? 

{data.map((contact) => {
            return (
              <ContactListItem
                key={contact._id}
                id={contact._id}
                name={contact.name}
              />
            );
          })}

File: components/StyledAppContainer/index.js
Comment (MariaRiosNavarro) [reply components/StyledAppContainer/index.js:1]: I use the span to be able to make styles if there is only one element inside (svg), the div is usually used when I have more things inside and bigger.

File: pages/contacts.js
Comment (MariaRiosNavarro) [reply pages/contacts.js:1]: I have not needed it, so I have not changed it.

Comment (MariaRiosNavarro) [reply pages/contacts.js:1]: I use the Hr as a placeholder to separate and organise my pages. I have already promised a coach that I will transfer his styles to a border-bottom when I am finishing. As it is now it serves as a visual aid on the page and in the code to recognise where I am, hence the name.

Comment (MariaRiosNavarro) [reply pages/contacts.js:1]: as a beginner I don't use the implicit return because it helps me to understand how my code works and to be more readable.

**Review by nastiche — APPROVED** [review]

**Pull request merged**